code/datums/dna.dm

```diff
@@ -112,6 +112,10 @@ GLOBAL_LIST_INIT(total_uf_len_by_block, populate_total_uf_len_by_block())
 	new_dna.species = new species.type
 	new_dna.species.species_traits = species.species_traits
 	new_dna.real_name = real_name
+	// Mutations aren't gc managed, but they still aren't templates
+	// Let's do a proper copy
+	for(var/datum/mutation/human/mutation in mutations)
+		new_dna.add_mutation(mutation, mutation.class, mutation.timeout)
 	new_dna.mutations = mutations.Copy()
```

Comment:
Left `mutations.Copy()` on the next line even though you're doing a proper copy

Also, `as anything`?

Reply:
Ah I'm dumb
IIRC the typecast is there for other loops because mutation/non_human
Don't wanna break the chain

Reply:
> `mutation/non_human`

pain

Reply:
WE MUST ALWAYS BE PREPARED FOR NON HUMAN MUTATIONS, DON'T YOU UNDERSTAND!!!?@w!?